packages/acceptance-tests/pkg-tests-core/sources/utils/config.yaml

```diff
@@ -0,0 +1,37 @@
+storage: /Users/jpicado/.local/share/verdaccio/storage_yarn
```

Comment:
I guess this won't work everywhere :p

Reply:
Hehe ... yeah the idea is create a temp folder and publish all fixture packages there on the setup phase. For now I am working in that part, I managed to publish but I don’t know how to invoke ‘yarn publish’ programmatically with the dev version. I did it using exec with npm ☺️) just for proving works my approach.